neurolib/models/multimodel/builder/adex.py

```diff
@@ -0,0 +1,944 @@
+"""
+Adaptive exponential integrate-and-fire mean-field approximation.
+
+Main reference:
+    Augustin, M., Ladenbauer, J., Baumann, F., & Obermayer, K. (2017).
+    Low-dimensional spike rate models derived from networks of adaptive
+    integrate-and-fire neurons: comparison and implementation. PLoS Comput Biol,
+    13(6), e1005545.
+
+Additional reference:
+    Cakan C., Obermayer K. (2020). Biophysically grounded mean-field models of
+    neural populations under electrical stimulation. PLoS Comput Biol, 16(4),
+    e1007822.
+"""
+
```

Comment:
```suggestion
```
I would remove this (like in all other model files).

Reply:
there are citations for relevant papers in all `MultiModel` model files :) but I can remove them, of course..